portfolio/src/main/webapp/style.css

```diff
@@ -33,6 +33,20 @@
 }
 
 #homelinks img {
-    width: 50px;
-    padding: 10px;
+  width: 50px;
+  padding: 10px;
+}
+
+#projectscontent {
+  margin-right: auto;
+  left: 10%;
+  position: relative;
+}
+#projectscontent .projectblock {
+  border: 1mm;
+  border-color: lightcoral;
```

Comment:
nit: generally prefer hex colors compared to the name colors (ie lightcoral = `#f08080`), assuming that there will be more colors being used across codebase, some of the hex-colors not having a corresponding name.